storage/src/device.rs

```diff
@@ -799,12 +799,19 @@ pub trait BlobObject: AsRawFd {
 ///
 /// All blob Io requests are actually served by the underlying [BlobCache] object. A new method
 /// [update()]() is added to switch the storage backend on demand.
-#[derive(Clone)]
 pub struct BlobDevice {
     //meta: ArcSwap<Arc<dyn BlobCache>>,
     blobs: ArcSwap<Vec<Arc<dyn BlobCache>>>,
     blob_count: usize,
 }
+impl Clone for BlobDevice {
```

Comment:
I am curious why the `arc-library` eliminated the Clone implementation but we can safely implement by ourselves?